helpers/utils.js

```diff
@@ -1,3 +1,29 @@
 export const getAcrFromIdToken = idToken => (
   JSON.parse(Buffer.from(idToken.split('.')[1], 'base64').toString('utf8')).acr
 );
+
+
+/**
+ * Permet de traduire un caractère en encodable pour le web " " -> "%20"
+ * @param {string} x character à encoder
+ */
+const percentialize = x => `%${x.charCodeAt(0).toString(16).toUpperCase()}`;
+/**
+ * Equivalent de la fonction encodeURIComponent mais compatible avec le norme RFC3986
+ * @param {string} s chaine de caractère à encoder pour le web
+ */
+const rfc3986EncodeURIComponent = s => encodeURIComponent(s).replace(/[!'()*]/g, percentialize);
+
+/**
+ * Permet de sérialiser un objet standard en paramètres d'URL
+ * @param {Object} params objet à transformer
+ */
+export const transformParams = params => Object.entries(params).map(([k, v]) => `${rfc3986EncodeURIComponent(k)}=${rfc3986EncodeURIComponent(v)}`).join('&');
+
+/**
+ * @description regex permettant de tester le texte de description d'erreur
+ * @example j\'ai trouvé + d\'une erreur & 200 cul-de-sac = un problème #grave \
```

Comment:
can we keep this in the test file. It didn't helped me to understand what this function do. Or perhaps use a simpler example here. For example, the real production example that was the cause of this PR. Plus, the `@franceconnect` appears highlighted in my IDE, which is confusing:

![image](https://user-images.githubusercontent.com/4190728/68234061-c72f7e80-0000-11ea-9f01-a1efc8aba071.png)